lib/fluent/plugin/exception_detector.rb

```diff
@@ -79,7 +79,7 @@ def self.supported
 
     PYTHON_RULES = [
       rule(:start_state, /^Traceback \(most recent call last\):$/, :python),
-      rule(:python, /^[\t ]+File /, :python_code),
+      rule([:python, :python_code], /^[\t ]+File /, :python_code),
```

Comment:
Do you want to change the [test](https://github.com/GoogleCloudPlatform/fluent-plugin-detect-exceptions/blob/HEAD/test/plugin/test_exception_detector.rb#L624-L626) as well?

Reply:
I added a second test case, which detects the traceback from #62